docs/tutorials/postgresql.ipynb

```diff
@@ -0,0 +1,368 @@
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "MUXex9ctTuDB"
+      },
+      "source": [
+        "## Setup and usage"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "upgCc3gXybsA",
+        "colab_type": "text"
+      },
+      "source": [
+        "### Install required tensorflow-io packages, and restart runtime"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "uUDYyMZRfkX4",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "%tensorflow_version 2.x \n",
+        "!pip install tensorflow-io-nightly"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "yZmI7l_GykcW",
+        "colab_type": "text"
+      },
+      "source": [
+        "### Install and setup PostgreSQL\n",
+        "\n",
+        "PostgreSQL database server is available on many platforms. In case it is not available, the following is needed to install PostgreSQL on Ubuntu:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "YUj0878jPyz7",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "!apt-get install -y -qq postgresql\n",
+        "\n",
+        "!service postgresql start"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Afb01zkACsuW",
+        "colab_type": "text"
+      },
+      "source": [
+        "PostgreSQL will setup a user with username `postgres` by default. However, this username is not assigned a password initially. We will need to update the database first. The following assigns password `postgres` to username `postgres`:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "3HHfVdv9D3Jv",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "!sudo -u postgres psql -U postgres -c \"ALTER USER postgres PASSWORD 'postgres';\""
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "9reCVv0mE_9O",
+        "colab_type": "text"
+      },
+      "source": [
+        "### Create database and prepare data\n",
+        "\n",
+        "For demo purposes we will create a database and populate the database with some data. The data we use are the stock prices of AAPL in 2019, downloaded from Yahoo Finance in [CSV format](https://github.com/tensorflow/io/blob/master/docs/tutorials/postgresql/aapl.csv). The following is a sneak preview of the CSV data:\n",
```

Comment:
I don't think I can publish it like this. 

Can you try to find a dataset with a clear open-license?

I believe anything from: https://archive.ics.uci.edu/ml/index.php is okay.